Documentation conflict - Drive limitations

CHDAFNI-MSFT opened the pull request:
The FAQs page lists additional drive limitations whereas the main page does now - https://docs.microsoft.com/en-us/azure/storage/storage-import-export-tool-preparing-hard-drives-import#faq

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

CHDAFNI-MSFT committed:
Documentation conflict - Drive limitations

The FAQs page lists additional drive limitations whereas the main page does now - https://docs.microsoft.com/en-us/azure/storage/storage-import-export-tool-preparing-hard-drives-import#faq
